tile/TileRTL_constant.py

```diff
@@ -0,0 +1,21 @@
+tile_port_direction_dict = {
```

Comment:
Remove this file and add the two dicts in https://github.com/tancheng/VectorCGRA/blob/master/lib/util/common.py? And use `PORT_xxx` as key?

Reply:
done